src/main/java/net/neoforged/neoforge/common/conditions/IConditionBuilder.java

```diff
@@ -44,27 +44,11 @@ default ICondition tagEmpty(TagKey<Item> tag) {
         return new TagEmptyCondition(tag.location());
     }
 
-    default ICondition isFeatureEnabled(FeatureFlagSet requiredFeatures) {
-        return FlagCondition.isEnabled(requiredFeatures);
+    default ICondition featureFlags(FeatureFlagSet requiredFeatures) {
```

Comment:
Same here as for the `FlagCondition` rename comment, not a fan maybe could be something like `featureFlagsEnabled` to show it passes when the given flags are enabled.
This would also follow the naming convention for other conditions like `modLoaded`